pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/go-logr/logr"
```

Comment:
Can't we use logger from `logger, ctx := ktesting.NewTestContext(t)` instead of this `go-logr` ?

Reply:
updated